web/templates/authorize.gohtml

```diff
@@ -73,6 +73,13 @@
                 </fieldset>
               </div>
             </div>
+            <div class="govuk-radios__divider">or</div>
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="unable-to-id" name="return-code" type="radio" value="X">
+              <label class="govuk-label govuk-radios__label" for="unable-to-id">
+                Unable to prove identity
+              </label>
+            </div>
```

Comment:
would be nice to have one for a scenario that won't lead to vouching, if we are going to be using return codes

Reply:
and maybe guard this stuff with an envvar so other users who don't care about these options won't see them